docs/admin_guide.md

```diff
@@ -443,7 +479,6 @@ Edit the main NGINX config file: `sudo nano /etc/nginx/nginx.conf` with the foll
 
 ```nginx
 ssl_protocols TLSv1.2 TLSv1.3; # Requires nginx >= 1.13.0 else only use TLSv1.2
-ssl_prefer_server_ciphers on;
```

Comment:
Yea you're right you most likely want to have this set to `off` (default is off).